Alignment/TrackerAlignment/plugins/TrackerSystematicMisalignments.cc

```diff
@@ -106,62 +106,59 @@ TrackerSystematicMisalignments::TrackerSystematicMisalignments(const edm::Parame
 
 void TrackerSystematicMisalignments::beginJob()
 {
-		
+
 }
 
 
 void TrackerSystematicMisalignments::analyze(const edm::Event& event, const edm::EventSetup& setup){
-	
+
 	//Retrieve tracker topology from geometry
 	edm::ESHandle<TrackerTopology> tTopoHandle;
 	setup.get<TrackerTopologyRcd>().get(tTopoHandle);
 	const TrackerTopology* const tTopo = tTopoHandle.product();
-	
+
 	edm::ESHandle<GeometricDet>  geom;
-	setup.get<IdealGeometryRecord>().get(geom);	 
+	setup.get<IdealGeometryRecord>().get(geom);
 	edm::ESHandle<PTrackerParameters> ptp;
 	setup.get<PTrackerParametersRcd>().get( ptp );
 	TrackerGeometry* tracker = TrackerGeomBuilderFromGeometricDet().build(&*geom, *ptp );
-	
+
 	//take geometry from DB or randomly generate geometry
 	if (m_fromDBGeom){
 		//build the tracker
 		edm::ESHandle<Alignments> alignments;
 		edm::ESHandle<AlignmentErrorsExtended> alignmentErrors;
-		
+
 		setup.get<TrackerAlignmentRcd>().get(alignments);
 		setup.get<TrackerAlignmentErrorExtendedRcd>().get(alignmentErrors);
-		
+
 		//apply the latest alignments
 		GeometryAligner aligner;
 		aligner.applyAlignments<TrackerGeometry>( &(*tracker), &(*alignments), &(*alignmentErrors), AlignTransform() );
-		
+
 	}
-	
+
 	theAlignableTracker = new AlignableTracker(&(*tracker), tTopo);
-	
+
 	applySystematicMisalignment( &(*theAlignableTracker) );
-	
+
 	// -------------- writing out to alignment record --------------
 	Alignments* myAlignments = theAlignableTracker->alignments() ;
-	AlignmentErrorsExtended* myAlignmentErrorsExtended = theAlignableTracker->alignmentErrors() ;
-	
+
 	// Store alignment[Error]s to DB
 	edm::Service<cond::service::PoolDBOutputService> poolDbService;
 	std::string theAlignRecordName = "TrackerAlignmentRcd";
-	std::string theErrorRecordName = "TrackerAlignmentErrorExtendedRcd";
-	
+
 	// Call service
 	if( !poolDbService.isAvailable() ) // Die if not available
 		throw cms::Exception("NotAvailable") << "PoolDBOutputService not available";
-	
+
 	poolDbService->writeOne<Alignments>(&(*myAlignments), poolDbService->beginOfTime(), theAlignRecordName);
-	poolDbService->writeOne<AlignmentErrorsExtended>(&(*myAlignmentErrorsExtended), poolDbService->beginOfTime(), theErrorRecordName);
 }
```

Comment:
The use case is exactly this:  take the APEs from the GT (or input sqlite file if you want) and copy them to the output sqlite file. Indeed it doesn't make sense to misalign the APE, but creating an APE payload that matches the misalignment you create makes a lot of sense to me. This functionality is not available but costs little effort to implement.